nvptx-as.c

```diff
@@ -897,9 +944,83 @@ fork_execute (const char *prog, char *const *argv)
   do_wait (prog, pex);
 }
 
+/* Determine if progname is available in PATH.  */
+static bool
+program_available (const char *progname)
```

Comment:
It is copied and modified from libiberty/make-relative-prefix.c.  It is sufficiently different that sharing the code would require big changes into libiberty.